storm-core/src/jvm/org/apache/storm/security/auth/plain/PlainClientCallbackHandler.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.storm.security.auth.plain;
+
+import org.apache.storm.security.auth.AbstractSaslClientCallbackHandler;
+
+public class PlainClientCallbackHandler extends AbstractSaslClientCallbackHandler {
+
+    /*
+     * For plain, using constants for a pair of user name and password.
+     */
+    public PlainClientCallbackHandler() {
+        _username = USERNAME;
```

Comment:
I forgot to ask before is the user this always reports to nimbus/etc now "username"?  If so I really would prefer to pass in the user name of the current user on this system, or even better have the server set the user to null like we were doing before.